python/langsmith/client.py

```diff
@@ -2710,6 +2710,13 @@ def create_dataset(
         inputs_schema: Optional[Dict[str, Any]] = None,
         outputs_schema: Optional[Dict[str, Any]] = None,
         metadata: Optional[dict] = None,
+        inputs: Optional[Sequence[Mapping[str, Any]]] = None,
```

Comment:
Thoughts on it just being examples=[]?

Then less ambiguity around examples_metadata or ids

Reply:
yea the metadata overlap ain't great. but having a totally different api than create_examples feels worse to me